### PR TITLE
nethogs 0.8.1 (new formula)

### PR DESCRIPTION
**Pull request by Rudloff**

### CODE REVIEW

**Comment by Rudloff** [comment]
Hello,

Here is formula for nethogs.
I first submitted it as a Linux-only formula: https://github.com/Homebrew/linuxbrew/pull/722
But then decided to port it to OSX.
This formula is based on the last stable release with the OSX patch applied.

File: Library/Formula/nethogs.rb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    end
+  end
+  # Add CPPFLAGS in Makefile
+  patch do
+    url "https://github.com/raboof/nethogs/commit/544a06b024eb4b24c807e77ae871c2c270cb74af.patch"
+    sha256 "8f0edf78be727c79fd933380a889156a01f25994acf05afb2aa6d6d1df8acac3"
+  end
+
+  def install
+    inreplace "Makefile", "prefix := /usr/local", "prefix := #{prefix}"
```
Comment (bfontaine) [inline Library/Formula/nethogs.rb]: Can’t you use e.g. `system "make", "install", "prefix=#{prefix}"`?

Comment (Rudloff) [inline Library/Formula/nethogs.rb]: You're right, thanks!

File: Library/Formula/nethogs.rb [hunk 2]
```diff
@@ -0,0 +1,32 @@
+class Nethogs < Formula
+  desc "Net top tool grouping bandwidth per process"
+  homepage "https://raboof.github.io/nethogs/"
+  url "https://github.com/raboof/nethogs/archive/v0.8.1.tar.gz"
+  sha256 "4c30ef43814549974a5b01fb1a94eb72ff08628c5a421085b1ce3bfe0524df42"
+
+  if OS.linux?
```
Comment (dunn) [inline Library/Formula/nethogs.rb]: The `OS.linux?`/`OS.mac?` modifications are added downstream by Linuxbrew; we don't have the time or resources to support Linux directly.  Please remove anything Linux-specific, thank you!

Comment (Rudloff) [inline Library/Formula/nethogs.rb]: Sorry, I didn't know that. I removed it.

**Comment by apjanke** [comment]
Looks good now.

Merged. Thank you for your contribution to Homebrew!

**Pull request closed without merging**